tests/Feature/Filament/SpotifySyncPageTest.php

```
@@ -0,0 +1,200 @@
+    public function it_displays_playlist_statistics(): void
+    {
+        /* Arrange */
+        $this->actingAs($this->user);
+
+        /* Act */
+        $response = Livewire::test(SpotifySync::class);
+
+        /* Assert */
+        $response->assertSee('total_playlists');
+        $response->assertSee('total_tracks');
+    }
```

PHPMD (2.15.0)

The method it_displays_playlist_statistics is not named in camelCase. (undefined)

(CamelCaseMethodName)